.maint/paper_author_list.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
 """Generate an author list for a new paper or abstract."""
+import json
```

Comment:
As @effigies suggested, let's separate style commits from this PR, so that they can be included/ignored altogether